README.md

```diff
@@ -38,6 +38,10 @@ The backend application is using the PERN Stack (Postgres, Express React, Node).
 
         brew install node@12
 
+- Install Lerna globally
+
+        npm i -g lerna
+
```

Comment:
I'd move this under the LTS comment below, which is about the Node install. Otherwise LGTM.

Reply:
requested changes made